.travis_centos.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# Install dependency packages and start mysqld service.
+sh .travis_setup_centos.sh
+
+# To avoid install error.
+sed -i '/eventmachine/d' Gemfile
```

Comment:
Why is this needed? Can't we fix the install error?

Reply:
Thanks for the comment.
It is to avoid install error by `bundle install` on CentOS.
But let me check again to fix it.

Reply:
The reason of the error is just because `g++` command was not installed on the environment.

Reply:
I fixed it and rebased.